Terminal-iOS/Login/IntroRemoteDataManager.swift

```diff
@@ -32,10 +33,35 @@ class IntroRemoteDataManager: IntroRemoteDataManagerProtocol {
         ]
         
         var result =  false
-        var url = URL(string: "http://3.35.154.27:3000/v1/user")!
+        let url = URL(string: "http://3.35.154.27:3000/v1/user")!
         AF.request(url, method: .post, parameters: params, encoding: JSONEncoding.default).responseJSON { response in
             result = JSON(response.data)["result"].bool!
         }.resume()
         return result
     }
+    
+    func getJoinValidInfo(joinMaterial: [String], completionHandler: @escaping (_ result: Bool, _ message: Any) -> ()) {
+        var params: Parameters = [
+            "email":"\(joinMaterial[0])",
+            "password":"\(joinMaterial[1])"
+        ]
+        var url = URL(string: "http://3.35.154.27:3000/v1/user/login")!
+        AF.request(url, method: .post, parameters: params, encoding: JSONEncoding.default).responseJSON { response in
+            var result = JSON(response.data)["result"].bool!
+            switch result {
+            case true:
+                print(JSON(response.data)["data"]["id"])
+                var data = JSON(response.data)["data"]["id"].int!
+                completionHandler(result, data)
+                break
+            case false:
+                print(JSON(response.data)["result"])
+                var data = JSON(response.data)["message"].string!
+                completionHandler(result, data)
+                break
+            }
```

Comment:
```suggestion
            switch response.result {
            case .success(let value):
                print(JSON(response.data)["data"]["id"])
                var data = JSON(response.data)["data"]["id"].int!
                completionHandler(result, data)
                break
            case .failure(let err):
                print(JSON(response.data)["result"])
                var data = JSON(response.data)["message"].string!
                completionHandler(result, data)
                break
            }
```

이런식이 명확 한거같은데 서버측에서 파라미터의 mismatch 가 있음에도 불구하고 .success로 떨어져서 실패했을시에 처리하기가 애매하군요 다시 서버측과 이야기해봐야겠어요

Reply:
네엡~~